contracts/cw20-stake/src/contract.rs

```diff
@@ -462,6 +465,34 @@ pub fn query_hooks(deps: Deps) -> StdResult<GetHooksResponse> {
     })
 }
 
+pub fn query_list_stakers(
+    deps: Deps,
+    start_after: Option<String>,
+    limit: Option<u32>,
```

Comment:
similar question here, would it be useful to have the option to query stakers at a particular height?